packages/react/src/utils/checkInternetAccess.ts

```diff
@@ -0,0 +1,30 @@
+import { Dispatch } from "redux";
+import { v4 as uuid } from "uuid";
+
+import { externalUrls } from "../constants/externalUrls";
+import { analyzeSolution } from "../store/actions/backend";
+import { setCurrentMessageUpdate } from "../store/actions/error";
+
+export const checkInternetAccess  = async (solutionPath: string, dispatch: Dispatch) => {
+  const haveInternet:boolean = await window.backend.checkInternetAccess();
+  if (!haveInternet) {
+    dispatch(
+      setCurrentMessageUpdate([{
+        messageId: uuid(),
+        groupId: "internetAccessFailed",
+        type: "error",
+        header: "No Internet Access",
```

Comment:
In case of proxy, would the internetCheck fail? Since we are checking on Nuget Package download. In that case do we want to add this detail in the message?

Reply:
updated to "unable to access s3", "Please check your internet connection" to be a more accurate message